server/conn.go

```diff
@@ -790,7 +790,7 @@ func (cc *clientConn) handleFieldList(sql string) (err error) {
 // resultsets, it's used to support the MULTI_RESULTS capability in mysql protocol.
 func (cc *clientConn) writeResultset(goCtx goctx.Context, rs ResultSet, binary bool, more bool) error {
 	defer terror.Call(rs.Close)
-	if rs.SupportChunk() {
+	if !cc.server.cfg.DisableChunk && rs.SupportChunk() {
```

Comment:
Could we check  DisableChunk  in SupportChunk?

Reply:
The ResultSet don't have the config to check.